content/en/Getting started/checklist.md

```diff
@@ -38,18 +38,9 @@ Under the Details tab, you can review:
 - [Cloud platform components](../details/#cloud-platform-components)
 - [Test data](../details/#test-data) sensitivity
 
-## Asset
-
-Under the Asset tab, you can review:
-
-- [Asset type](../assets/asset-type/)
-- [Description](../assets/asset-description/)
-- [Documentation](../assets/asset-description/#asset-documentation)
-
 ## Reports
 
-{{% pentest-report-requirements %}}
-<!-- For content, see the following file: layouts/shortcodes/pentest-report-requirements.html -->
+<p>{{% min-credits-report %}}.</p>
```

Comment:
@mjang I tested this formatting several times on the localhost, but it doesn't render properly in the deploy preview. Any ideas?

Reply:
TLDR, this is not a "blocker".

I've tried a few "quickie" suggestions without success, based on
- https://stackoverflow.com/questions/69031332/hugo-shortcode-adding-extra-space-at-end-of-link-text
- https://discourse.gohugo.io/t/white-space-issue-in-markdown/25258/2

I think it might be a Goldmark/Black Friday Markdown issue (see second bullet), but haven't solved the problem yet. Can you open a (internal) Jira?